Filters/exceptions.txt

```diff
@@ -1,3 +1,10 @@
+! Prevent possible blocking of AdGuard DNS by included third-party filters
+@@||adguard.io^
+@@||dns.adguard.com^
```

Comment:
As far as I can see, it redirects to `adguard-dns.io`, so maybe it should be also added?
By the way, shouldn't these rules end with `|`?
I mean:
```adblock
@@||dns.adguard.com^|
```

Reply:
Will fix in https://github.com/AdguardTeam/HostlistsRegistry

Reply:
adguard-dns.io - only dashboard. This filter blocks DNS servers.